src/components/viewAllPage/allSubjectsPage/components/SubjectsColumn.scss

```diff
@@ -44,12 +47,20 @@
       }
       @include tabletL {
         text-align: center;
-        width: 24%;
-        padding-left: 0;
+        width: 16vw;
         font-size: 1.3vw;
         margin: 0.8vh 0.5%;
         border-radius: 0.8vw;
-        display: inline-block;
+      }
+      &:hover {
+        background: var(--white) !important;
+        color: var(--theme-dark-blue) !important;
+        // .round-circle {
+        //   background: white !important;
+        // }
```

Comment:
just remove it if not needed no need for comments

Reply:
ok